<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTagger</class>
 <widget class="QMainWindow" name="FileTagger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FileTagger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>896</width>
      <height>450</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="CREATE">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QListWidget" name="PREVIOUS_TAGS">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>100</y>
        <width>201</width>
        <height>111</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="filename">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>661</width>
        <height>25</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="similar_Tags">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>110</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Similar Tags </string>
      </property>
     </widget>
     <widget class="QLabel" name="newtags">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enters Tags </string>
      </property>
     </widget>
     <widget class="QPushButton" name="ADD_TAG">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>70</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>SAVE TAGS</string>
      </property>
     </widget>
     <widget class="QListWidget" name="history">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>631</width>
        <height>181</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
     <widget class="QLabel" name="HISTORY">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>HISTORY</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ENTER_TAG">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>491</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="filenametag">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>File/Folder/website path</string>
      </property>
     </widget>
     <widget class="QLabel" name="filenametag_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Drag &amp; Drop File/Folder </string>
      </property>
     </widget>
     <widget class="QLabel" name="Versiono">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>320</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Version no  -:</string>
      </property>
     </widget>
     <widget class="QLabel" name="version">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>320</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>290</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>390</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>300</y>
        <width>20</width>
        <height>101</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>300</y>
        <width>20</width>
        <height>101</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>(separated by </string>
      </property>
     </widget>
     <widget class="QPushButton" name="updates">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>350</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Check Updates</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>140</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>(Press enter </string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>160</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>to auto complete)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>blank space)</string>
      </property>
     </widget>
     <zorder>ENTER_TAG</zorder>
     <zorder>PREVIOUS_TAGS</zorder>
     <zorder>filename</zorder>
     <zorder>similar_Tags</zorder>
     <zorder>newtags</zorder>
     <zorder>ADD_TAG</zorder>
     <zorder>history</zorder>
     <zorder>HISTORY</zorder>
     <zorder>filenametag</zorder>
     <zorder>filenametag_2</zorder>
     <zorder>Versiono</zorder>
     <zorder>version</zorder>
     <zorder>line</zorder>
     <zorder>line_2</zorder>
     <zorder>line_3</zorder>
     <zorder>line_4</zorder>
     <zorder>label_2</zorder>
     <zorder>updates</zorder>
     <zorder>label_4</zorder>
     <zorder>label</zorder>
     <zorder>label_5</zorder>
    </widget>
    <widget class="QWidget" name="FIND">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QLineEdit" name="ENTER_TAG_FIND">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>591</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="ENTERTAGLABEL">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Enters Tags</string>
      </property>
     </widget>
     <widget class="QListWidget" name="FILE_LIST">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>761</width>
        <height>311</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="autoScroll">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="FILELIST_LABEL">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>801</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>FILES/FOLDERS/WEBSITES   - SORTED BY APPROXIMATE STRING MATCHING ALGORITHM   (CLICK TO OPEN)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="REMOVETAG">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>140</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>REMOVE TAGS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CHECKFILES">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>280</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>CHECK for 
 missing FILES</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>181</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>(separated by blank space)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="virtualFS">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>40</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CREATE 
 Virtual Folder</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="AproximateradioButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>APPROXIMATE STRING MATCHING</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="ExactradioButton2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>40</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>EXACT STRING MATCHING</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="ANY">
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>10</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ANY</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>filename</tabstop>
  <tabstop>ENTER_TAG</tabstop>
  <tabstop>ADD_TAG</tabstop>
  <tabstop>FILE_LIST</tabstop>
  <tabstop>ENTER_TAG_FIND</tabstop>
  <tabstop>history</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>PREVIOUS_TAGS</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
